spec/features/items/edit_spec.rb

```diff
@@ -16,7 +16,7 @@
         expect(current_path).to eq("/items/#{@tire.id}/edit")
         expect(page).to have_link("Gatorskins")
         expect(find_field('Name').value).to eq "Gatorskins"
-        expect(find_field('Price').value).to eq '$100.00'
+        expect(find_field('Price').value).to eq '100'
```

Comment:
I realized that in the forms, you shouldn't prepopulate numeric fields with non-numeric values (i.e., $100.00) because the form won't accept it

Reply:
Yes. The test from my experience should reflect the DB input